app/src/main/java/de/perflyst/untis/notification/NotificationReceiver.java

```diff
@@ -28,6 +32,14 @@ public void onReceive(Context context, final Intent intent) {
 			}
 
 		final NotificationManager notificationManager = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+		if (notificationManager != null && Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
`notificationManager != null &&` is somewhat useless, because `context.getSystemService(Context.NOTIFICATION_SERVICE)` never returns null, but it an android studio warning